packages/office-ui-fabric-react/src/components/Rating/examples/Rating.Basic.Example.tsx

```diff
@@ -12,38 +12,48 @@ export class RatingBasicExample extends React.Component<any, any> {
     super();
 
     this.state = {
-      rating: null
+      largeStarRating: null,
+      smallStarRating: null,
+      tenStarRating: null
     };
   }
 
   public render() {
     return (
       <div className='ms-RatingBasicExample'>
+        Large Stars:
         <Rating
+          id={ 'largeRatingStar' }
           min={ 1 }
           max={ 5 }
-          rating={ this.state.rating }
-          onChanged={ this._onChanged }
+          size={ RatingSize.Large }
+          rating={ this.state.largeStarRating }
+          ariaLabelId={ this._getRatingComponentAriaLabel(this.state.largeStarRating, 5) }
```

Comment:
this is weird, the props wants the id of an existing element and you're passing in the string? Did you want to add something like 

`getAriaLabel?: (rating: number, min: number, max: number) => string` 

to the Rating Props, and then use that to set the `aria-label`?

Reply:
In that case we will end up doing same thing as earlier. Where I have used rating , min and max to set the string.
as given below. Also, In the project I didn't find any resources file where I can define these static strings. So, I have used the current existing approach.
**let rating: number = this.state.rating ? this.state.rating : 0; 
 return 'Rating value is ' + rating + ' of ' + this.props.max**

Reply:
I don't quite understand, `ariaLabelId` expects the `id` of an element that describes the one the attribute is applied to. Do you want to set `ariaLabel`?

Reply:
Let me take this offline with you.

Reply:
The component expects the `id` of an element and you are passing in a label string, that doesn't work.

Reply:
Updated with getAriaLabel callback function